WalletWasabi.Fluent/Views/Wallets/Home/History/Columns/IndicatorsColumnView.axaml

```diff
@@ -19,20 +19,12 @@
       </Button>
     </Panel>
 
-    <Panel>
-      <!-- Confirmed -->
-      <PathIcon IsVisible="{Binding IsConfirmed}"
-                Data="{StaticResource checkmark_filled}"
-                Height="11"
-                Opacity="0.6"
-                ToolTip.Tip="{Binding ConfirmedToolTip}" />
-      <!-- Pending -->
-      <PathIcon IsVisible="{Binding !IsConfirmed}"
-                Data="{StaticResource clock_regular}"
-                Height="14"
-                Opacity="0.6"
-                ToolTip.Tip="Pending" />
-    </Panel>
```

Comment:
@SuperJMN 

If you take a look at the height of the icons, you can see they are not the same. This is one thing why the solution is not good, the second reason is we should avoid storing resource names as a string on ViewModel level.

Could you do something similar like:
https://github.com/zkSNACKs/WalletWasabi/blob/755ddc6d99ed26180e80d9a35969282b3c0b6b14/WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml#L102-L105